net/iperf3/src/opnsense/mvc/app/models/OPNsense/Iperf/Menu/Menu.xml

```diff
@@ -0,0 +1,7 @@
+<menu>
+  <Interfaces>
+    <Diagnostics>
+      <iperf3 url="/ui/iperf/" />
```

Comment:
lower case? may be better without "3"

Reply:
I write the tool like on the cli. You may be right but there may be other versions of iperf as well.

Reply:
Not for us. The "iperf3" issue is a FreeBSD naming silliness, it still has "iperf" which is iperf2 and that's it.